cypress/e2e/swap.test.ts

```diff
@@ -1,13 +1,57 @@
+import { SupportedChainId, WETH9 } from '@uniswap/sdk-core'
+
+import { getTestSelector } from '../utils'
+
 describe('Swap', () => {
+  const verifyAmount = (field: 'input' | 'output', amountText: string | null) => {
+    if (amountText === null) {
+      cy.get(`#swap-currency-${field} .token-amount-input`).should('not.have.value')
+    } else {
+      cy.get(`#swap-currency-${field} .token-amount-input`).should('have.value', amountText)
+    }
+  }
+
+  const verifyToken = (field: 'input' | 'output', tokenSymbol: string | null) => {
+    if (tokenSymbol === null) {
+      cy.get(`#swap-currency-${field} .token-symbol-container`).should('contain.text', 'Select token')
+    } else {
+      cy.get(`#swap-currency-${field} .token-symbol-container`).should('contain.text', tokenSymbol)
+    }
+  }
+
+  const selectOutput = (tokenSymbol: string) => {
+    // open token selector...
+    cy.contains('Select token').click()
+    // select token...
+    cy.contains(tokenSymbol).click({ force: true })
+
+    cy.get('body')
+      .then(($body) => {
+        if ($body.find(getTestSelector('TokenSafetyWrapper')).length) {
+          return 'I understand'
```

Comment:
this is not a selector, its the text content

Reply:
i'll rename